pkg/repositories/api_key_repository.go

```diff
@@ -10,11 +10,13 @@ import (
 )
 
 type ApiKeyRepository interface {
-	Create(ctx context.Context, tx db.Transaction, apiKey persistence.ApiKey) (persistence.ApiKey, error)
+	Create(ctx context.Context, apiKey persistence.ApiKey) (persistence.ApiKey, error)
```

Comment:
As we don't create a user and the API key at the same time anymore we don't need the transaction.